src/main/java/com/cupid/jikting/member/entity/Member.java

```diff
@@ -70,4 +71,11 @@ public void validatePassword(PasswordEncoder passwordEncoder, String password) {
     public void updatePassword(String newPassword) {
         this.password = newPassword;
     }
+
+    public void blockCompanies() {
+        if(memberCompanies.isEmpty()) {
```

Comment:
p1. 소괄호 앞 공백 추가해주세요!

```suggestion
        if (memberCompanies.isEmpty()) {
```